Makefile

```diff
@@ -27,6 +28,11 @@ help:
 	@echo "    clean-cluster:   kill the kind cluster only"
 	@echo "    rpm/deb:         generate rpm or debian packages"
 
+publish: build-containers
+	hack/publish_containers.sh
```

Comment:
publish_containers.sh content is not big, maybe just put it directly in here?

Reply:
yeah, I was not sure about the for loop from bash in the makefile. I'd need to unroll it, may be it could be done. Anyway for CI it could potentially be different (using the existing actions)

Thanks.